.github/workflows/build_with_conda.yml

```diff
@@ -16,6 +16,12 @@ jobs:
       matrix:
         include:
```

Comment:
Could you add a comment on why C++ and Python tests are done separately?

Reply:
yes, its faster since both these things can run in parallel.
To run the python tests, we do not need to compile the c++ tests. Ie we save the time used for compiling , so about 20 minutes.
And in the c++ test run, we do not need to run the python tests

Reply:
It might be helpful to add the explanation to the file itself.